CRM/Odoosync/Sync/Contact/SyncInformation.php

```diff
@@ -1,91 +1,112 @@
 <?php
 
 /**
- * Gets appropriate contact for synchronization with Odoo
+ * Updates contact sync information when sync successful/error
```

Comment:
I think CRM_Odoosync_Sync_Contact_SyncInformation  is not an accurate name for this class, the class only handles contact sync post processing, so I think a better name would be : 

**CRM_Odoosync_Sync_Contact_ResponseHandler** and I would rename updateContactSuccessfulSync method to handleSuccess and updateContactErrorSync to handleError